x-pack/plugins/canvas/common/lib/fonts.js

```diff
@@ -8,7 +8,10 @@ export const americanTypewriter = {
   label: 'American Typewriter',
   value: `'American Typewriter', 'Courier New', Courier, Monaco, mono`,
 };
-export const arial = { label: 'Arial', value: `Arial, sans-serif` };
+export const arial = {
+  label: 'Arial',
+  value: `Arial, sans-serif`,
```

Comment:
Avoid changing files that are not germane to your pull request.  It appears this file was simply formatted.  If so, include as a separate PR.